engine/src/coretextfonts.cpp

```diff
@@ -32,6 +32,48 @@
 #import <ApplicationServices/ApplicationServices.h>
 #endif
 
+#ifdef TARGET_SUBPLATFORM_IPHONE
+// Stored as a global variable
+static MCArrayRef s_font_map = nil;
+
+// Populate the VariableArray s_font_map
+// from the input mapping list (<mapped name>=<PostScript name>).
+// Each <PostScript name> is storred keyed as <mapped name>
+void add_ios_fontmap(const char* p_mapping)
+{
+    if (s_font_map == nil)
+    {
+        if (!MCArrayCreateMutable(s_font_map))
+            return;
+    }
+
+    MCAutoStringRef t_mapping;
+    if (!MCStringCreateWithCString(p_mapping, &t_mapping))
+        return false;
+
+    uindex_t t_separator;
+    if (!MCStringFirstIndexOfChar(*t_mapping, '=', 0, kMCStringOptionCompareExact, t_separator))
+        return;
+    
+    MCAutoStringRef t_from, t_to;
+    MCNewAutoNameRef t_from_as_name;
+    if (!MCStringCopySubstring(*t_mapping, MCRangeMake(0, t_separator), &t_from)
+            || !MCNameCreate(*t_from, &t_from_as_name)
+            || !MCStringCopySubstring(*t_mapping, MCRangeMake(t_separator, UINDEX_MAX), &t_to))
```

Comment:
I think this includes the '=' in the t_to string doesn't it? I presume that is not wanted.